appveyor.yml

```diff
@@ -47,55 +47,44 @@ platform:
 install:
   - ps: Install-Product node $env:nodejs_version $env:platform
   # upgrade npm
-  - npm install -g npm@6
-  - npm install -g @bahmutov/print-env@1
+  - choco install yarn
```

Comment:
why choco?

Reply:
Ease to get yarn installed is the only reason. Should I investigate an alternative mechanism or manually install yarn? I can manually download the MSI or use Scoop as an alternative. If there is no preference, then I'd prefer using something like choco or scoop; similarly to how we use apt-get for Debian Linux distros, etc.

Reply:
what about `npm install -g yarn` ?

Reply:
I noticed that yarn [mentions this](https://yarnpkg.com/lang/en/docs/install/#alternatives-stable) is not recommended. With that said, I am uncertain of the impact of installing it via npm. The reason for the recommendation to not do so may not impact us.

Reply:
ok, that's fair, and it looks like appveyor has choco installed by default so we're good there. nice :+1: